src/app/chat/views/communities.nim

```diff
@@ -367,10 +378,21 @@ QtObject:
     except Exception as e:
       error "Error requesting to join the community", msg = e.msg
 
+  proc removeMembershipRequest(self: CommunitiesView, requestId: string, accepted: bool) =
+    var i = 0
+    for request in self.myCommunityRequests:
+      if (request.id == requestId):
+        self.myCommunityRequests.delete(i, i)
```

Comment:
Does this work? I think Nim doesn't like when we delete elements from  a `seq` inside a loop on that `seq`. 
What I had to do in the past is store the indexes in another `seq` and then another loop to delete.